### PR TITLE
Docs register

### PR DESCRIPTION
**Pull request by mvn-haphan-dn**
## Description
Docs register
## Reference Requirements
- Task ID: #125

## Reference Isuees
#{issue_id}

## Migrations
YES | NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

**Comment by at-haytran** [comment]
LGTM

File: docs/api-references/register.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+#### Response - Success
+| Field | Type | Description |
+|-------|------|-------------|
+| user | Object | Object user |
+| id | Integer | Id of user |
+| email | String | Email of user |
+| password | Integer | Password of user  |
+| identity_number | Integer | Identity number of user |
+| language | String | Language of user |
```
Comment (at-quangtran) [inline docs/api-references/register.md]: response ko thấy trường này

File: docs/api-references/register.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+### `POST` Register
+
+```
+/api/register
```
Comment (at-quangtran) [inline docs/api-references/register.md]: /api/users/register

Comment (at-haphan) [reply docs/api-references/register.md]: /api/register cũng được đúng không ạ? Tại login bọn em cũng dùng là /api/login chứ k có users đây route ạ

**Review by mvn-quangtran-dn — APPROVED** [review]
LGTM

**Pull request merged**